### ISSUE TITLE
Release 8.4:  libignition-transport8-log-dev fails to install

### ISSUE THREAD

**Issue by mikaelarguedas** [gazebo-release/gz-transport8-release#10]
Since November 18th installation of gazebo fails to install libignition-transport8-log-dev that has some conflict with libignition-transport8-core-dev.

```
Preparing to unpack .../397-libignition-transport8-log-dev_8.4.0-2~focal_amd64.deb ...
Unpacking libignition-transport8-log-dev:amd64 (8.4.0-2~focal) ...
dpkg: error processing archive /tmp/apt-dpkg-install-KP9rba/397-libignition-transport8-log-dev_8.4.0-2~focal_amd64.deb (--unpack):
 trying to overwrite '/usr/include/ignition/transport8/gz/transport/log.hh', which is also in package libignition-transport8-core-dev:amd64 8.4.0-2~focal
```
Full log available here: https://github.com/osrf/docker_images/actions/runs/3501113487/jobs/5864423277#step:5:4154

**Comment by peci1** [comment]
This also happens on bionic.

**Comment by peci1** [comment]
This is the third time I remember some ign update got broken. Would it be possible to add some pre-release tests to the buildfarm which could test that it is possible to update from previous binary install to the new one?

**Comment by j-rivero** [comment]
> This is the third time I remember some ign update got broken. Would it be possible to add some pre-release tests to the buildfarm which could test that it is possible to update from previous binary install to the new one?

Yes, we have been using the autopkgtest feature in Debian packaging to avoid this to happen. For ignition-transport8 here there was a combination of a missing feature, https://github.com/gazebo-release/gz-transport8-release/pull/8 and the ign to gz rename transition. This old releases are usually pretty stable but renaming have been affecting all them, sorry for the inconvenience. I'm going to release the fix in 8.4.0-3.

**Comment by j-rivero** [comment]
> the autopkgtest feature in Debian packaging

This is how it usually acts, by installing all the packages in the build https://build.osrfoundation.org/job/ign-transport8-debbuilder/125/console#console-section-1

**Comment by j-rivero** [comment]
Fixed in release `8.4.0-3`.
